localization/en.ftl

```diff
@@ -275,16 +275,21 @@ multiline-name = Multiline Expressions
 multiline-desc = Splits expressions onto multiple lines to better make use of available space.
 multiline-opt-widthBeforeMultiline-name = Width Threshold (%)
 multiline-opt-widthBeforeMultiline-desc = Minimum width (as a percent of the viewport size) at which point wrapping will occur. On mobile, this value is tripled.
-multiline-opt-automaticallyMultilinify-name = Insert Linebreaks while Typing
-multiline-opt-automaticallyMultilinify-desc = Automatically splits expressions onto multiple lines while you type, bypassing the need to use Ctrl+M.
+multiline-opt-alterLayoutWhileTyping-name = Alter Layout while Typing
```

Comment:
"Update" would be a little clearer than "Alter", here and below

Reply:
Def agreed. Will change.